tests/drivers/candev/main.c

```diff
@@ -275,7 +273,8 @@ int main(void)
     can_init(&periph_dev, &(candev_conf[0]));
     candev = &(periph_dev.candev);
 #if defined(BOARD_SAME54_XPRO)
-    gpio_init(AT6561_STBY_PIN, AT6561_STBY_MODE);
+    gpio_init(AT6561_STBY_PIN, GPIO_OUT);
+    gpio_clear(AT6561_STBY_PIN);
```

Comment:
I think the driver should handle this, not the app.

Reply:
i think there are good reason to keep this out of the driver -> either in the app ( but with a more generic name) or in the board

Reply:
No, the driver should set this when `CANOPT_STATE_OFF`, `CANOPT_STATE_SLEEP` or `CANOPT_STATE_LISTEN_ONLY` is set via `CANOPT_STATE`.

Reply:
But we have those power states in the CAN driver exactly for that.
The pin belongs to the CAN controller, so only the CAN controller should control it.

Reply:
But we don't add them to the `can_t` structure, right?. I would agree with adding the handling of the standby pin of the CAN transceiver only if we keep it defined in the board itself and not including it as new member to the CAN controller structure

Reply:
I don't see what's the problem with adding a `gpio_t standby_pin` to `can_conf_t`

Reply:
Because that will open the door to adding always other pins from any other CAN transceivers in the market that are offering more functionalities than just standby mode. The `can_conf_t` should have only the pins that the CAN controller as a hardware unit provides I believe, that's why I would prefer keeping the CAN transceiver pins just in the board definition

Reply:
What other can transceivers are there that provide more than a standby pin?

Reply:
![image](https://github.com/RIOT-OS/RIOT/assets/53952217/5d291860-dd72-4d09-be4b-2a7e8d3368f4)
This is an example, same product provider but two different versions of the CAN transceiver. AT6561 is the one used for the SAME54-xpro

Reply:
if we need such abstraction it has to go on top of the can driver (?sys/can?)  it needs a struct like {can_dev_t dev, trasceiver_t transceiver}

or 

just an enable that goes into the board init